lib/windows/launcher/actions/desktopShortcutActions.js

```diff
@@ -132,305 +151,51 @@ function createShortcutForLinux(app) {
-            			<string>Viewer</string>
-            		</dict>
-            	</array>
-            	<key>CFBundleExecutable</key>
-            	<string>Application Stub</string>
-            	<key>CFBundleIconFile</key>
-            	<string>icon.icns</string>
-            	<key>CFBundleIdentifier</key>
-            	<string>${identifier}${app.isOfficial ? '' : '-local'}</string>
-            	<key>CFBundleInfoDictionaryVersion</key>
-            	<string>6.0</string>
-            	<key>CFBundleName</key>
-            	<string>${fileName}</string>
-            	<key>CFBundlePackageType</key>
-            	<string>APPL</string>
-            	<key>CFBundleShortVersionString</key>
-            	<string>1.2</string>
-            	<key>CFBundleSignature</key>
-            	<string>????</string>
-            	<key>CFBundleSupportedPlatforms</key>
-            	<array>
-            		<string>MacOSX</string>
-            	</array>
-            	<key>CFBundleURLTypes</key>
-            	<array/>
-            	<key>CFBundleVersion</key>
-            	<string>419</string>
-            	<key>DTCompiler</key>
-            	<string>com.apple.compilers.llvm.clang.1_0</string>
-            	<key>DTPlatformBuild</key>
-            	<string>7C43</string>
-            	<key>DTPlatformVersion</key>
-            	<string>GM</string>
-            	<key>DTSDKBuild</key>
-            	<string>15E8</string>
-            	<key>DTSDKName</key>
-            	<string>macosx10.11internal</string>
-            	<key>DTXcode</key>
-            	<string>0720</string>
-            	<key>DTXcodeBuild</key>
-            	<string>7C43</string>
-            	<key>LSMinimumSystemVersion</key>
-            	<string>10.5</string>
-            	<key>LSMinimumSystemVersionByArchitecture</key>
-            	<dict>
-            		<key>x86_64</key>
-            		<string>10.6</string>
-            	</dict>
-            	<key>LSUIElement</key>
-            	<true/>
-            	<key>NSAppleScriptEnabled</key>
-            	<string>YES</string>
-            	<key>NSMainNibFile</key>
-            	<string>ApplicationStub</string>
-            	<key>NSPrincipalClass</key>
-            	<string>NSApplication</string>
-            	<key>NSServices</key>
-            	<array/>
-            	<key>UTExportedTypeDeclarations</key>
-            	<array/>
-            	<key>UTImportedTypeDeclarations</key>
-            	<array/>
-            </dict>
-            </plist>
-        `;
-        const wflowPath = path.join(tmpAppTemplatePath, '/Contents/document.wflow');
+        const tmpAppTemplatePath = path.join(config.getTmpDir(), '/com.nordicsemi.nrfconnet/template.app');
+
+        // Create Info.plist
+        const infoTemplatePath = path.join(appTemplatePath, '/Contents/Info.plist');
+        const infoTmpPath = path.join(tmpAppTemplatePath, '/Contents/Info.plist');
+        const identifier = `com.nordicsemi.nrfconnet.${app.name}${app.isOfficial ? '' : '-local'}`;
```

Comment:
Typo: `nrfconnect`

Reply:
Fixed

Reply:
Still seeing "nrfconnet" instead of "nrfconnect" here.

Reply:
Oh, there is another one.
Fixed